crates/iceberg/src/spec/table_properties.rs

```diff
@@ -137,6 +162,21 @@ impl TableProperties {
     pub const PROPERTY_WRITE_TARGET_FILE_SIZE_BYTES: &str = "write.target-file-size-bytes";
     /// Default target file size
     pub const PROPERTY_WRITE_TARGET_FILE_SIZE_BYTES_DEFAULT: usize = 512 * 1024 * 1024; // 512 MB
+
+    /// Compression codec for metadata files (JSON)
+    pub const PROPERTY_METADATA_COMPRESSION_CODEC: &str = "write.metadata.compression-codec";
+    /// Default metadata compression codec - uncompressed
+    pub const PROPERTY_METADATA_COMPRESSION_CODEC_DEFAULT: &str = "uncompressed";
```

Comment:
```suggestion
    /// Default metadata compression codec - uncompressed
    pub const PROPERTY_METADATA_COMPRESSION_CODEC_DEFAULT: &str = "none";
```
this should be none according to 
https://iceberg.apache.org/docs/nightly/configuration/#write-properties

<img width="769" height="68" alt="Image" src="https://github.com/user-attachments/assets/75c8885b-4653-43ce-978b-6573646f6e69" />